### PR TITLE
MODKBEKBJ-340 - Custom labels: retrieve custom labels list

### PR DESCRIPTION
**Pull request by natali-zaitseva**
## Purpose
Implementation for https://issues.folio.org/browse/MODKBEKBJ-340 for addition of custom labels functionality defined https://docs.google.com/presentation/d/1WYCM9D7MzD32nNzoJ1kXfVDSTRJHaTxW5nVKoAtAiIs/edit#slide=id.p

## Approach
* updated ModuleDescriptor with additional endpoints for custom labels
* created custom-labels raml file to define endpoints
* implemented GET /custom-labels collection
* added tests

## Learning
* Ruby implementation - https://github.com/folio-org/mod-kb-ebsco/pull/86
* RM API - https://developer.ebsco.com/interactive/holdingsiq#/

### CODE REVIEW

File: ramls/types/resources/resourcePutDataAttributes.json [hunk 1]
```diff
@@ -110,6 +110,26 @@
         "$ref": "../coverage.json"
       }
     },
+    "userDefinedField1": {
+      "type": "string",
```
Comment (dmtkachenko) [inline ramls/types/resources/resourcePutDataAttributes.json]: correct me if im wrong, the fields have length limitation, don't they?
do we want to apply it here (maxLength)?

File: ramls/providers.raml [hunk 2]
```diff
@@ -123,35 +123,32 @@ traits:
...
+                  strict: false
+                  value: !include examples/providers/providers_providerId_packages_get_400_response.json
+          404:
+            description: Not Found
+            body:
+              application/vnd.api+json:
+                type: jsonapiError
+                example:
+                  strict: false
+                  value: !include examples/providers/providers_providerId_packages_get_404_response.json
```
Comment (carolegodfrey) [inline ramls/providers.raml]: Are there any changes to this file?

File: ramls/types/resources/resourcePutDataAttributes.json (hunk 1 above)
Comment (natali-zaitseva) [reply ramls/types/resources/resourcePutDataAttributes.json]: yes, `maxLength` restriction should be added. I thought that custom label value length was covered in other PR. I will update json files for length restriction. Thanks for review @dmtkachenko @carolegodfrey

File: ramls/providers.raml (hunk 2 above)
Comment (natali-zaitseva) [reply ramls/providers.raml]: @carolegodfrey just formatted the text here

**Pull request merged**